### ISSUE TITLE
Recommendation of aria-controls for the aria-expanded attribute

### ISSUE THREAD

**Issue by Wildebrew** [w3c/aria#1376]
The ARIA 1.2 definition of aria-expanded contains the following recommendation "If a grouping container that can be expanded or collapsed is not owned by the element that has the aria-expanded attribute, the author SHOULD identify the controlling relationship by referencing the container from the element that has aria-expanded with the aria-controls property.".
I find this a bit odd. A grouping element should never be owned by a control (e.g. a button or a checkbox) so this is barely ever the case. Maybe for gridcells or treeitems, but other than that this is rare.
This basically recommends use of aria-controls on all controls that have aria-expanded. Given the lack of support for the attribute, often coupled with unnecessary verbosity, using aria-controls can make for a worse user experience.
I would drop recommendation for use of aria-controls if
* The grouping element whose display is controlled by the element with aria-controls is a sibling of the control element and if the grouping element immedaitely follows the control in content order.
* Therefore a button with aria-expanded followed by the content whose display is controlled by activating the button does not need aria-controls, while a button that controls content elsewhere on the page does need it.
* I admit this is not a high priority issue, but I have recommended against use of aria-controls except where the content controlled by an element is not immediately adjacent to it.

**Comment by JAWS-test** [comment]
See: https://github.com/w3c/aria/issues/995

**Comment by mcking65** [comment]
@Wildebrew 
Birkir, Could you please add your thoughts to #995 so we can have a single thread on this topic?

**Comment by jnurthen** [comment]
Closing as a duplicate of #995 - please add the text if not covered
